### PR TITLE
docs: Fix typos in Intro to Determined

### PR DESCRIPTION
**Pull request by tara-det-ai**
Fix typos and other errors in the Intro to Determined.

### CODE REVIEW

**Review by rb-determined-ai — APPROVED** [review]

File: docs/architecture/introduction.rst [hunk 1]
```diff
@@ -100,11 +100,10 @@ of CLI commands.
 Model Registry
 ==============
 
-The Model Registry is a way to group together conceptually related checkpoints (including ones
-across different experiments), storing metadata and longform notes about a model, and retrieving the
-latest version of a model for use or futher development. The Model Registry can be accessed through
-the WebUI, Python SDK, REST API, or CLI, though the WebUI has some features that the others are
-missing.
+The Model Registry is a way to group conceptually-related checkpoints (including ones across
```
Comment (dzhu) [inline docs/architecture/introduction.rst:103]:
```suggestion
The Model Registry is a way to group conceptually related checkpoints (including ones across
```
https://www.merriam-webster.com/words-at-play/6-common-hypercorrections-and-how-to-avoid-them/hyphenating-ly-adverbs

File: docs/architecture/introduction.rst [hunk 2]
```diff
@@ -100,11 +100,10 @@ of CLI commands.
...
+different experiments), store metadata and long form notes about a model, and retrieve the latest
```
Comment (dzhu) [inline docs/architecture/introduction.rst:104]:
```suggestion
different experiments), store metadata and long-form notes about a model, and retrieve the latest
```

File: docs/architecture/introduction.rst [hunk 3]
```diff
@@ -367,25 +366,26 @@ If you are using static resource pools and launching agents by hand, you will ne
...
-``resource_manager`` and ``resource_pools``. The ``resource_manager`` section is for cluster level
-setting such as which pools should be used by default and the default scheduler settings. The
-``scheduler`` information is identical to the ``scheduler`` field in the legacy format. The
-``resource_pools`` section is a list of resource pools each of which has a name, description and
-resource pool level settings. Each resource pool can be configured with a ``provider`` field that
-contains the same information as the ``provisioner`` field in the legacy format. Each resource pool
-can also have a ``scheduler`` field that sets resource pool specific scheduler settings. If the
-``scheduler`` field is not set for a specific resource pool, the default settings are used.
+Resource pools were introduced with Determined 0.14.0 resulting in changes to the
+:ref:`master-config-reference` format.
```
Comment (dzhu) [inline docs/architecture/introduction.rst:370]:
```suggestion
:ref:`master configuration <master-config-reference>` format.
```

File: docs/architecture/introduction.rst [hunk 4]
```diff
@@ -367,25 +366,26 @@ If you are using static resource pools and launching agents by hand, you will ne
...
-and provisioner settings for the cluster. The new format has the top level fields
-``resource_manager`` and ``resource_pools``. The ``resource_manager`` section is for cluster level
-setting such as which pools should be used by default and the default scheduler settings. The
-``scheduler`` information is identical to the ``scheduler`` field in the legacy format. The
-``resource_pools`` section is a list of resource pools each of which has a name, description and
-resource pool level settings. Each resource pool can be configured with a ``provider`` field that
-contains the same information as the ``provisioner`` field in the legacy format. Each resource pool
-can also have a ``scheduler`` field that sets resource pool specific scheduler settings. If the
-``scheduler`` field is not set for a specific resource pool, the default settings are used.
+Resource pools were introduced with Determined 0.14.0 resulting in changes to the
```
Comment (dzhu) [inline docs/architecture/introduction.rst:369]:
```suggestion
Resource pools were introduced with Determined 0.14.0, resulting in changes to the
```

File: docs/architecture/introduction.rst (hunk 3 above)
Comment (dzhu) [reply docs/architecture/introduction.rst:370]: Could you also update the link to `agent-config-reference` above similarly?

File: docs/architecture/introduction.rst [hunk 5]
```diff
@@ -367,25 +366,26 @@ If you are using static resource pools and launching agents by hand, you will ne
...
+:ref:`master-config-reference` format.
+
+Since the change is backwards-compatible, cluster configurations that use earlier formats (prior to
+Determined 0.14.0) will still work. A configuration in the earlier format is interpreted as a
+cluster with a single resource pool that is the default for both CPU and GPU tasks. To take full
+advantage of resource pools, convert to the latest format. Converting is a simple process of moving
+around and renaming a small number of top-level fields.
+
+The earlier format had the top level fields of ``scheduler`` and ``provisioner`` which set the
```
Comment (dzhu) [inline docs/architecture/introduction.rst:378]:
```suggestion
The earlier format had the top-level fields of ``scheduler`` and ``provisioner``, which set the
```

**Review by dzhu — APPROVED** [review]

File: docs/architecture/introduction.rst (hunk 1 above)
Comment (tara-det-ai) [reply docs/architecture/introduction.rst:103]: resolved

File: docs/architecture/introduction.rst (hunk 2 above)
Comment (tara-det-ai) [reply docs/architecture/introduction.rst:104]: resolved

File: docs/architecture/introduction.rst (hunk 4 above)
Comment (tara-det-ai) [reply docs/architecture/introduction.rst:369]: resolved

File: docs/architecture/introduction.rst (hunk 3 above)
Comment (tara-det-ai) [reply docs/architecture/introduction.rst:370]: done

**Pull request merged**